ocaml/driver/main_args.ml

```diff
@@ -1630,6 +1638,7 @@ struct
     mk_no_alias_deps F._no_alias_deps;
     mk_app_funct F._app_funct;
     mk_no_app_funct F._no_app_funct;
+    mk_directory F._directory;
```

Comment:
I don't think we need this in ocamldoc, which is basically deprecated.

Reply:
Removed!